fineract-core/src/main/java/org/apache/fineract/infrastructure/core/service/DateUtils.java

```diff
@@ -306,6 +306,12 @@ public static boolean isDateInThePast(final LocalDate localDate) {
         return isBeforeBusinessDate(localDate);
     }
 
+    public static boolean occursOnDayFromAndIncludingAndUpTo(final LocalDate fromInclusive, final LocalDate upToNotInclusive,
```

Comment:
We already have this: `isDateInRangeFromExclusiveToInclusive`, and i like the naming strategy of this more.
Can you please rename to `isDateInRangeFromInclusiveToExclusive` ?